fe/fe-core/src/main/java/org/apache/doris/analysis/ColumnDef.java

```diff
@@ -61,10 +61,19 @@ public class ColumnDef {
     public static class DefaultValue {
         public boolean isSet;
         public String value;
+        // only used for datetime column type
+        public boolean isCurrentTimestamp;
 
         public DefaultValue(boolean isSet, String value) {
             this.isSet = isSet;
             this.value = value;
+            this.isCurrentTimestamp = false;
```

Comment:
Do not use boolean current_timestamp here. Use a expr to save it. Actually, currentTimestamp is function "now". we may add other default value types. Expr is more normal because it could stand for any functions.

Reply:
when user use a function as default value. We save the function name to value, and the actual function to the expr field.